web/modules/custom/itk_openid_connect_interpreter/src/Helper/Helper.php

```diff
@@ -2,27 +2,40 @@
 
 namespace Drupal\itk_openid_connect_interpreter\Helper;
 
-use Drupal\Core\Routing\RouteMatchInterface;
+use Drupal\Component\Utility\Crypt;
+use Drupal\Core\Session\AccountInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
 
 /**
  * A helper.
  */
 class Helper {
+  use StringTranslationTrait;
+
   /**
-   * Constructor.
+   * Implements hook_openid_connect_userinfo_alter().
+   *
+   * @todo Make it generic and reusable across multiple sites?
    */
-  public function __construct(
-    private readonly RouteMatchInterface $routeMatch,
-  ) {
+  public function openidConnectUserinfoAlter(array &$userinfo, array $context) {
+    // @todo
+    $encryptedUuid = Crypt::hashBase64('abc');
```

Comment:
Use something form `$userinfo` here, e.g.

```suggestion
    $encryptedUuid = Crypt::hashBase64($userinfo['uuid']);
```